### ISSUE TITLE
Texture Sizes

### ISSUE THREAD

**Issue by crazytimtimtim** [Stratosphere-Studios/777-300ER#14]
Some textures are way to big, here are some that should be reduced @nathroxer 

- landing gear should be 2k
- aft overhead should also be 2k
- clocks should be 2k but maybe even less
- trim indicators - I noticed you have these absolutely huge files and just cut the scale in half in planemaker. What's the point? Just cut the size of the texture in half
- trim indicator elements - why are the small elements like the green band or the pointer the same size as the big textures? Just make the texture the size of the element...

VRAM doesn't grow on trees!!!

**Comment by IASXP** [comment]
However, some specimens of vram grow on bushes.

**Comment by crazytimtimtim** [comment]
This is true, however, those specimens are extremely rare and only found near a limited number of extremely hard to find plots of land.

**Comment by Matt726-S** [comment]
> This is true, however, those specimens are extremely rare and only found near a limited number of extremely expensive and hard to find plots of land.

The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

**Comment by IASXP** [comment]
> > This is true, however, those specimens are extremely rare and only found near a limited number of extremely expensive and hard to find plots of land.
> 
> The FitnessGram™ Pacer Test is a multistage aerobic capacity test that progressively gets more difficult as it continues. The 20 meter pacer test will begin in 30 seconds. Line up at the start. The running speed starts slowly, but gets faster each minute after you hear this signal. [beep] A single lap should be completed each time you hear this sound. [ding] Remember to run in a straight line, and run as long as possible. The second time you fail to complete a lap before the sound, your test is over. The test will begin on the word start. On your mark, get ready, start.

While the FitnessGram pacer test may be 20 meters, in truth if you are the average obese American, this is a daunting task. From my experience many Americans should take with them 5 gallons of Coke along with 10 pounds of chips to properly complete this mammoth task without withrdawl symptoms. Even then, they MUST take breaks to concentrate on eating their food for the best results.

**Comment by crazytimtimtim** [comment]
> > This is true, however, those specimens are extremely rare and only found near a limited number of extremely expensive and hard to find plots of land.
> 
> Your statement is simply not true. I personally have a vram farm in my backyard, and so do many including nathroxer. You should plant some vram bushes, and you will never have to complain about vram ever. In case you are wondering, plant [this](https://www.ebay.com/itm/111888818427?) in the ground. The linked selection is the cheapest vram seeds money can buy, though if you have more money I would recommend the micron sub-species. One important thing to note when buying vram seeds is the pH factor. The pH should never be under 6 but if you MUST then 7 is also fine. Also I like to feed the seeds with uranium, which gives the plant nice glowing factors. I personally source my uranium from the Chernobyl plant, however extracting urainum from urainum glass on [ebay](https://www.ebay.com/itm/324802090026?) also gives good results. lastly I like to top it off with a little bit of high quality soil so that it will keep growing without letting you down. Here is the [link](https://www.youtube.com/watch?v=xvFZjo5PgG0).

@nathroxer, Can you tell be a bit about your vram farm?
